src/Config.php

```diff
@@ -225,7 +227,7 @@ public function setTypePreference(string $type, string $preference, ?string $con
     public function setAlias(string $name, string $class) : self
     {
         if (! class_exists($class) && ! interface_exists($class)) {
-            throw new Exception\ClassNotFoundException('Could not find class "' . $class . '"');
+            throw new Exception\ClassNotFoundException($class);
```

Comment:
This is a nitpick spotted while working on this. This call duplicates the message because ClassNotFoundException only expects the class name.